test-it/test.js

```diff
@@ -238,6 +238,14 @@
         vireo.eggShell.setPrintFunction(function (text) {
             testOutput += text + '\n';
         });
+        // Vireo unit tests can't fire JS events, so register no-op registration functions
+        vireo.controlEvents.setRegisterForControlEventsFunction(function () {
```

Comment:
I think that if we change the internal functions in `module_controlEvents` to not throw, then we wouldn't need to write these lines and avoid the linting rules. Unless we want our clients to explicitly opt-out of this functionality, as it is right now, then the other question would be; do we want the same opt-out behavior for the property node?